app/controllers/admin/custom_organisations_controller.rb

```diff
@@ -9,11 +9,17 @@ def create
     @custom_organisation = CustomOrganisationName.new(custom_org_params)
 
     if @custom_organisation.save
-      flash[:notice] = "Custom organisation has been successfully added"
+      flash.now[:notice] = "Custom organisation has been successfully added"
     end
     render :index
   end
 
+  def destroy
+    custom_org = CustomOrganisationName.find_by(params.fetch(:id))
```

Comment:
This won't work. There is `.find` and `.find_by`. `.find_by` requires a key, `.find` only takes an ID: https://api.rubyonrails.org/v5.2.2/classes/ActiveRecord/FinderMethods.html#method-i-find_by